signature/src/policy/mod.rs

```diff
@@ -27,6 +28,8 @@ pub enum ErrorInfo {
     ErrUnknowPolicyReqType,
     #[strum(to_string = "The reference match policy type name is Unknown.")]
     ErrUnknownMatchPolicyType,
+    #[strum(to_string = "The signature scheme is Unsupported.")]
+    ErrUnknownScheme,
```

Comment:
This is slightly confusing as the error text doesn't really align with the error ("unknown" compared to "unsupported").